### PR TITLE
refactor: update delete output

### PR DESCRIPTION
**Pull request by PatrickMenoti**
**WHAT**
- Use new centralized output for delete commands;

### CODE REVIEW

**Review by PabloDiehl — APPROVED** [review]

File: pkg/cmd/delete/edge_application/edge_application_test.go [hunk 1]
```diff
@@ -101,6 +101,6 @@ func TestCreate(t *testing.T) {
 		_, err := cmd.ExecuteC()
 		require.NoError(t, err)
 
-		assert.Equal(t, "Remote resources deleted successfully\n\n", stdout.String())
+		assert.Equal(t, "Remote resources deleted successfully\n", stdout.String())
```
Comment (maxwelbm) [inline pkg/cmd/delete/edge_application/edge_application_test.go:104]: Use the message variable to centralize maintenance in tests

File: pkg/cmd/delete/origin/origin_test.go [hunk 2]
```diff
@@ -28,7 +28,7 @@ func TestCreate(t *testing.T) {
 
 		_, err := cmd.ExecuteC()
 		require.NoError(t, err)
-		assert.Equal(t, "🚀 Origin 58755fef-e830-4ea4-b9e0-6481f1ef496d was successfully deleted\n\n", stdout.String())
+		assert.Equal(t, "Origin 58755fef-e830-4ea4-b9e0-6481f1ef496d was successfully deleted\n", stdout.String())
```
Comment (maxwelbm) [inline pkg/cmd/delete/origin/origin_test.go:31]: Use the message variable to centralize maintenance in tests

File: pkg/cmd/delete/variables/variables_test.go [hunk 3]
```diff
@@ -29,7 +29,7 @@ func TestCreate(t *testing.T) {
 		_, err := cmd.ExecuteC()
 		require.NoError(t, err)
 
-		assert.Equal(t, "🚀 Variable 7a187044-4a00-4a4a-93ed-d230900421f3 was successfully deleted\n\n", stdout.String())
+		assert.Equal(t, "Variable 7a187044-4a00-4a4a-93ed-d230900421f3 was successfully deleted\n", stdout.String())
```
Comment (maxwelbm) [inline pkg/cmd/delete/variables/variables_test.go:32]: Use the message variable to centralize maintenance in tests

Comment (maxwelbm) [inline pkg/cmd/delete/variables/variables_test.go:32]: remove \n to leave the next line formatting in the output method and be centered leaving only the message

File: pkg/cmd/delete/edge_application/edge_application_test.go (hunk 1 above)
Comment (maxwelbm) [reply pkg/cmd/delete/edge_application/edge_application_test.go:104]: remove \n to leave the next line formatting in the output method and be centered leaving only the message

File: pkg/cmd/delete/variables/variables_test.go (hunk 3 above)
Comment (maxwelbm) [reply pkg/cmd/delete/variables/variables_test.go:32]: remove \n to leave the next line formatting in the output method and be centered leaving only the message

Comment (PatrickMenoti) [reply pkg/cmd/delete/variables/variables_test.go:32]: Sure. I'll add a new line to the output method

**Review by maxwelbm — APPROVED** [review]

**Review by PabloDiehl — APPROVED** [review]

**Pull request merged**